packages/datetime/src/dateRangeInput.tsx

```diff
@@ -329,6 +332,12 @@ export class DateRangeInput extends AbstractComponent<IDateRangeInputProps, IDat
         this.setState(nextState);
     }
 
+    protected validateProps(props: IDateRangeInputProps & { children?: React.ReactNode }) {
```

Comment:
safe to remove `children`, esp considering that this component doesn't really support `children` 😄